test/tab-size.js

```diff
@@ -0,0 +1,373 @@
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'less\', tabSize:0}', function() {
+        var options = {syntax: 'less', tabSize: 0};
+
+        it('should return 15 for non-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringNonIndented, options);
+            assert.equal(startCol, 15);
+        });
+
+        it('should return 5 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringTabIndented, options);
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'less\', tabSize:1}', function() {
+        var options = {syntax: 'less', tabSize: 1};
+
+        it('should return 15 for non-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringNonIndented, options);
+            assert.equal(startCol, 15);
+        });
+
+        it('should return 2 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringTabIndented, options);
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'less\', tabSize:2}', function() {
+        var options = {syntax: 'less', tabSize: 2};
+
+        it('should return 15 for non-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringNonIndented, options);
+            assert.equal(startCol, 15);
+        });
+
+        it('should return 3 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringTabIndented, options);
+            assert.equal(startCol, 3);
+        });
+    });
+
+    describe('{syntax:\'less\', tabSize:4}', function() {
+        var options = {syntax: 'less', tabSize: 4};
+
+        it('should return 15 for non-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringNonIndented, options);
+            assert.equal(startCol, 15);
+        });
+
+        it('should return 5 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(lessStringTabIndented, options);
+            assert.equal(startCol, 5);
+        });
+    });
+
+    describe('{syntax:\'sass\'}', function() {
+        var options = {syntax: 'sass'};
+
+        it('should return 2 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(sassStringTabIndented, options);
+            assert.equal(startCol, 2);
+        });
+    });
+
+    describe('{syntax:\'sass\', tabSize:0}', function() {
+        var options = {syntax: 'sass', tabSize: 0};
+
+        it('should return 5 for tab-indented declaration', function() {
+            var startCol = getColorDeclarationStartCol(sassStringTabIndented, options);
+            assert.equal(startCol, 2);
```

Comment:
If you see the Saas String:
`var sassStringTabIndented = '$color:blue\na\n\tcolor:$color';`
You will see that the only indented part is the `color` declaration. It has been indented using a single `\t`.

With regards the `{syntax:'sass'}` test: no `tabSize` option has been set, then `tabSize` option should be set to 1. Thus  the value of the `color` declaration start column should be 2 which is correct.

With regards the `{syntax:'sass', tabSize:0}` test: tabSize` option has been set to 0, then `tabSize` option should be set to 1. Thus  the value of the `color` declaration start column should be 2 which is correct.